src/boss/cloudvm/worker_config_template.go

```diff
@@ -32,6 +32,15 @@ func LoadWorkerConfigTemplate(templatePath string, workerPath string) error {
 		return fmt.Errorf("failed to load template.json: %v", err)
 	}
 
+	// Patch worker-specific paths
```

Comment:
Hmmm, this is a bit of a problem.  Because what if they actually want to specify a different zygotes file, or something like that?

I'm not sure of the best way to handle this.  Thoughts?

One idea: if the field is empty in the template, we take that to mean we're supposed to patch it in.  But if it is not empty, we assume the user meant to specify something, and it should be left alone.

If we do override something, we should probably print something so it is less likely to go unnoticed.  I could see this being a point of annoyance for users who are trying to set these fields.